test/acceptance/authz/objects_validate_test.go

```diff
@@ -0,0 +1,141 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package test
+
+import (
+	"testing"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/google/uuid"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/weaviate/weaviate/client/authz"
+	"github.com/weaviate/weaviate/client/objects"
+	clschema "github.com/weaviate/weaviate/client/schema"
+	"github.com/weaviate/weaviate/entities/models"
+	"github.com/weaviate/weaviate/entities/schema"
+	"github.com/weaviate/weaviate/test/helper"
+	"github.com/weaviate/weaviate/usecases/auth/authorization"
+)
+
+func TestAuthZObjectValidate(t *testing.T) {
+	adminKey := "admin-key"
+	adminAuth := helper.CreateAuth(adminKey)
+	customUser := "custom-user"
+	customAuth := helper.CreateAuth("custom-key")
+
+	readDataAction := authorization.ReadObjectsCollection
+	readSchemaAction := authorization.ReadCollections
+
+	helper.SetupClient("127.0.0.1:8081")
+	defer helper.ResetClient()
```

Comment:
**Suggestion:** Since this test is co-located with the other testcontainers tests, are you willing to use testcontainers here too? Copying the pre-amble in the other tests would suffice as I'll be moving it all into some `func setup` method anyone in a PR soon. wydt?

Reply:
> are you willing to use testcontainers here too

not while we are developing 😓